### ISSUE TITLE
Current day isn't accessibly displayed in the calendar grid

### ISSUE THREAD

**Issue by GVodyanov** [nextcloud/calendar#7933]
### Steps to reproduce

The current day is only indicated with color in the calendar grid, this is not accessible.

### Expected behavior

Set `aria-current=true` on current day

### Actual behavior

Only color indicator

### Web server

None

### Database engine version

None

### PHP engine version

None

### Updated from an older installed version or fresh install

None

### List of activated apps

```Text

```

### Nextcloud configuration

```Text

```

### Web server error log

```Text

```

### Log file

```Text

```

### Browser log

```Text

```
